<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dagrapport</class>
 <widget class="QDialog" name="Dagrapport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dagrapport</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icons/report.png</normaloff>:/icons/icons/report.png</iconset>
  </property>
  <widget class="QDateEdit" name="dtpDatum">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dddd dd/MM/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>75</y>
     <width>66</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Afdrukken</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/printer.png</normaloff>:/icons/icons/printer.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>5</y>
     <width>66</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Afsluiten</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/stop2.png</normaloff>:/icons/icons/stop2.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresch">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>145</y>
     <width>66</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Verversen</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/refresh_all.png</normaloff>:/icons/icons/refresh_all.png</iconset>
   </property>
  </widget>
  <widget class="QTableView" name="tvPerProduct">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>676</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
    </font>
   </property>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
  </widget>
  <widget class="QTableView" name="tvPerBTW">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>230</y>
     <width>676</width>
     <height>66</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
    </font>
   </property>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="lbAantalVerkopen">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>15</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Aantal verkopen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbTotaal">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>tbTotaal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbTotaal">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>305</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Totaal:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tvKasboek">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbTotUit">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>575</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbTotIn">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>575</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbKastotalen">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>580</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kasboek totalen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbTicketTotaal">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>300</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>tbTicketTotaal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbTicketTotaal">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>305</y>
     <width>196</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ticket Totaal betaald:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tvUitgaven">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
    </font>
   </property>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="lbUigaven">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>330</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Uigaven:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbKasboek">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>330</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kasboek;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFacturatie">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>525</y>
     <width>66</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Facturatie van de daginkomsten</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/document_index.png</normaloff>:/icons/icons/document_index.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>66</width>
     <height>61</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Correctie betaalmiddelen</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/credit.png</normaloff>:/icons/icons/credit.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
